axelrod/fingerprint.py

```diff
@@ -327,7 +156,7 @@ def plot(self, col_map='seismic', interpolation='none', title=None,
             http://matplotlib.org/examples/images_contours_and_fields/interpolation_methods.html
         title : str, optional
             A title for the plot
-        colorbar : bool, optional
+        color_bar : bool, optional
```

Comment:
Change this back to `colorbar`. This is inline with `matplotlib`'s variable/method name (which is why the variable was called this to begin with) and this also introduces backwards incompatibility. 

Before I look through further please revert any variable names/function names (that are not relevant to this refactor) back to what they were. Having to re explain the reasoning behind every previous choice is just going to be too time consuming. Once you feel this PR is a simple refactor of the methods to functions let me know and I'll review.